composer.json

```diff
@@ -46,7 +47,7 @@
 	"require": {
 		"php": ">=5.5.0",
 		"composer/installers": "1.*,>=1.0.1",
-		"mediawiki/semantic-media-wiki": "~2.1|~3.0",
+		"mediawiki/semantic-media-wiki": "~2.1",
```

Comment:
Agree to being more energetic. Or just release what's available (as long as there are no catastrophic bugs).